Feature/45 basic authentication

subigre opened the pull request:
Add support for using EHRbase with basic authentication

ppazos commented:
@jsachau  can you please check the conflicts with develop branch?

jsachau commented:
@ppazos I have merged the newest develop branch

ppazos commented on src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:
```diff
@@ -278,15 +278,8 @@ public void createObservationUsingDefaultProfile() {
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Default profile is not supported for Observation. One of the following profiles is expected: [http://hl7.org/fhir/StructureDefinition/bodytemp, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/FiO2, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, http://hl7.org/fhir/StructureDefinition/heartrate, https://www.netzwerk-univ ...
```
@jsachau can you please use the multiline layout? it is easier to read.

jsachau replied:
@ppazos yes i did.

ppazos commented on src/test/java/org/ehrbase/fhirbridge/FhirBridgeApplicationIT.java:
```diff
@@ -299,15 +292,8 @@ public void createObservationUsingUnsupportedProfile() {
-            + "https://www.medizininformatik-initiative.de/fhir/core/StructureDefinition/ObservationLab, "
-            + "https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/sofa-score]",
-            OperationOutcomeUtil.getFirstIssueDetails(context, exception.getOperationOutcome()));
+                "Profile http://hl7.org/fhir/StructureDefinition/vitalsigns is not supported for Observation. One of the following profiles is expected: [http://hl7.org/fhir/StructureDefinition/bodytemp, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/FiO2, https://www.netzwerk-universitaetsmedizin.de/fhir/StructureDefinition/blood-pressure, http://hl7.org/fhir/StructureDefi ...
```
@jsachau can you please use the multiline layout? it is easier to read.

jsachau replied:
@ppazos there too. Sorry!

jsachau commented:
@ppazos I have updated the code a few days ago. The requested change is outdated since then.

subigre committed:
Implementation of issue #45

subigre committed:
Start EHRbase using basic auth

wlad committed:
Finalizes 'invalid blood pressure profil' test migration

also
- updates RF to v4.0 beta
- removes not-ready tag from Observation test suite

wlad committed:
Merge branch 'develop' into feature/NUM-544_Invalid-blood-pressure-tests-final-refactoring

wlad committed:
fix rebot call

wlad committed:
Merge branch 'feature/NUM-544_Invalid-blood-pressure-tests-final-refactoring' into feature/45_Enable-basic-auth-in-robot-tests

wlad committed:
Enables Basic Auth in Robot tests

wlad committed:
restors accidentially removed KW

wlad reviewed (approved):
looks good 👍
